tests/test_features/test_scatter.py

```diff
@@ -0,0 +1,188 @@
+        """
+        Assert that only two features are allowed for this visualizer in init
+        """
+        features = ["temperature", "relative_humidity", "light"]
+
+        with self.assertRaises(YellowbrickValueError) as context:
+            visualizer = ScatterViz(features=features)
+
+    def test_scatter_requires_two_features_in_numpy_matrix(self):
+        """
+        Assert that only two features are allowed for this visualizer if not
+        set in the init
+        """
+        visualizer = ScatterViz()
+        with self.assertRaises(YellowbrickValueError) as context:
+            visualizer.fit_transform(self.X, self.y)
+            self.assertTrue(
+                'only accepts two features' in str(context.exception))
+
+    def test_integrated_scatter(self):
+        """
+        Test scatter on the real, occupancy data set
+        """
+        # Load the data from the fixture
+        X = self.occupancy[[
+            "temperature", "relative_humidity", "light", "C02", "humidity"
+        ]]
+        y = self.occupancy['occupancy'].astype(int)
+
+        # Convert X to an ndarray
+        X = X.view((float, len(X.dtype.names)))
+
+        # Test the visualizer
+        features = ["temperature", "relative_humidity"]
+        visualizer = ScatterViz(features=features)
+        visualizer.fit_transform_poof(X[:, :2], y)
+
+    def test_scatter_quick_method(self):
+        """
+        Test scatter on the real, occupancy data set
+        """
+        # Load the data from the fixture
+        X = self.occupancy[[
+            "temperature", "relative_humidity", "light", "C02", "humidity"
+        ]]
+        y = self.occupancy['occupancy'].astype(int)
+
+        # Convert X to an ndarray
+        X = X.view((float, len(X.dtype.names)))
+
+        # Test the visualizer
+        features = ["temperature", "relative_humidity"]
+        ax = scatterviz(X[:, :2], y=y, ax=None, features=features)
+
+        # test that is returns a matplotlib obj with axes
+        self.assertIn('Axes', str(ax.properties()['axes']))
+
+    @unittest.skipUnless(pandas is not None,
+                         "Pandas is not installed, could not run test.")
+    def test_integrated_scatter_with_pandas(self):
+        """
+        Test scatter on the real, occupancy data set with a pandas dataframe
+        """
+        # Load the data from the fixture
+        X = self.occupancy[[
+            "temperature", "relative_humidity", "light", "C02", "humidity"
+        ]]
+        y = self.occupancy['occupancy'].astype(int)
+
+        # Convert X to a pandas dataframe
+        X = pandas.DataFrame(X)
+        X.columns = [
+            "temperature", "relative_humidity", "light", "C02", "humidity"
+        ]
+
+        # Test the visualizer
+        features = ["temperature", "relative_humidity"]
+        visualizer = ScatterViz(features=features)
+        visualizer.fit_transform_poof(X, y)
+
+    # def test_integrated_scatter_numpy_named_arrays(self):
```

Comment:
Not a big deal, but you could also decorate with @unittest.skip("not implemented yet") instead of commenting out the function. The benefit of the skip is that the tests will show the skipped test but not fail, alerting us that we have to implement the test sometime in the future.